examples/p4est_3d_dgsem/elixir_mhdmultiion_convergence.jl

```diff
@@ -0,0 +1,207 @@
+
+using OrdinaryDiffEqLowStorageRK
+using Trixi
+
+# 3D version of multi-ion MHD convergence test from
+#   - Rueda-Ramírez, A. M., Sikstel, A., & Gassner, G. J. (2025). An entropy-stable discontinuous Galerkin 
+#     discretization of the ideal multi-ion magnetohydrodynamics system. Journal of Computational Physics, 523, 113655.
+
+###############################################################################
+"""
+  electron_pressure_alpha(u, equations::IdealGlmMhdMultiIonEquations3D)
+Returns a fraction (alpha) of the total ion pressure for the electron pressure.
```

Comment:
```suggestion
Returns a fraction (alpha) of the total ion pressure for the electron pressure.

```
I think we normally have an empty line here